### PR TITLE
add 12CHARS to private domains

### PR DESCRIPTION
**Pull request by yahesh**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)
* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
  * [x] This request was _not_ submitted with the objective of working around other third-party limits
  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding.

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

  * [x] _Yes, I understand._ I could break my organization's website cookies etc. and the rollback timing, etc is acceptable. _Proceed._

---

Description of Organization
====

12CHARS is a start-up currently building a SaaS solution for cryptographically secured and reproducible decision-making in large(r) organisations. We are still in stealth mode but already prepare the security of the multi-tenant product as we try to enter the market with proper ISO 27001 processes already in place.

I represent 12CHARS as the founder of the organisation.

Organization Website: https://12chars.com

Reason for PSL Inclusion
====

12CHARS will assign each customer of the SaaS solution an individual sub-domain below the `12chars.dev` development domain and the `12chars.pro` production domain. Furthermore, 12CHARS hosts different internal and external services below the `12chars.it` internal IT domain. Through the PSL inclusion the cookies of the different customers and internal/external services shall be properly separated for security reasons.

The registration of the `12chars.it` internal IT domain is set to auto-renew. Unfortunately, the registration duration of *.it domains is limited to 1 year (see _"When does a domain expire?"_ in the [nic.it FAQ](https://www.nic.it/en/find-your-it/faq)).

DNS Verification via dig
=======

```
dig +short TXT _psl.12chars.dev
"https://github.com/publicsuffix/list/pull/1915"
```

```
dig +short TXT _psl.12chars.it
"https://github.com/publicsuffix/list/pull/1915"
```

```
dig +short TXT _psl.12chars.pro
"https://github.com/publicsuffix/list/pull/1915"
```

Results of Syntax Checker (`make test`)
=========

The syntax checker ran successfully:
```
============================================================================
Testsuite summary for libpsl 0.21.2
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11209,6 +11209,12 @@ zuerich
 // ===BEGIN PRIVATE DOMAINS===
 // (Note: these are in alphabetical order by company name)
 
+// 12CHARS: https://12chars.com
+// Submitted by Kenny Niehage <kenny@12chars.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:11213]: If possible, replace this with a role based address so it stays reachable when you leave the company.

**Review by simon-friedberger — APPROVED** [review]
* [x] Expiration
  * [x] 12chars.dev expires 2026-05-21
  * [x] 12chars.it has an expiry exception because of .it policy
  * [x] 12chars.pro expires 2026-05-21
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

File: public_suffix_list.dat (hunk 1 above)
Comment (yahesh) [reply public_suffix_list.dat]: We have implemented a PSL-specific e-mail address to implement this suggestion.

**Review by dnsguru — APPROVED** [review]
* Sorted to Guidelines
* DNS TXT = Pull Request URL
* Tests Pass
* No conflict with base

APPROVED (https://github.com/publicsuffix/list/pull/1915#event-11651748574) by @simon-friedberger

**Pull request merged**